tests/test_ocs_basic_install.py

```diff
@@ -44,8 +45,16 @@ def run(**kwargs):
     if not cluster_path:
         cluster_path = os.path.join(cluster_dir_parent, cluster_name)
     run_cmd(f"mkdir -p {cluster_path}")
-    pull_secret_path = os.path.join(templating.TOP_DIR, "data", "pull-secret")
-    with open(pull_secret_path, "r") as f:
+    pull_secret_file = os.path.join(templating.TOP_DIR, "data", "pull-secret")
```

Comment:
Would it be worth creating a function that retrieves the pull secret now that there is more than one filepath to check? I also think there were changes to how we read the pull-secret in #133  that we need to consider before making this change.